index.html

```diff
@@ -2290,6 +2290,277 @@ <h2>Definition of Roles</h2>
+          </tr>
+          <tr>
+            <th class="role-scope-head" scope="row">Required Context Role:</th>
+            <td class="role-scope"><rref>list</rref></td>
+          </tr>
+          <tr>
+            <th class="role-mustcontain-head" scope="row">Required Owned Elements:</th>
+            <td class="role-mustcontain"> </td>
+          </tr>
+          <tr>
+            <th class="role-required-properties-head">Required States and Properties:</th>
+            <td class="role-required-properties"> </td>
+          </tr>
+          <tr>
+            <th class="role-properties-head" scope="row">Supported States and Properties:</th>
+            <td class="role-properties">
+              <ul>
+                <li><pref>aria-level</pref></li>
+                <li><pref>aria-posinset</pref></li>
+                <li><pref>aria-setsize</pref></li>
+              </ul>
+            </td>
+          </tr>
+          <tr>
+            <th class="role-inherited-head" scope="row">Inherited States and Properties:</th>
+            <td class="role-inherited">Placeholder</td>
+          </tr>
+          <tr>
+            <th class="role-namefrom-head" scope="row">Name From:</th>
+            <td class="role-namefrom">author</td>
+          </tr>
+          <tr>
+            <th class="role-namerequired-head" scope="row">Accessible Name Required:</th>
+            <td class="role-namerequired"> </td>
+          </tr>
+          <tr>
+            <th class="role-namerequired-inherited-head" scope="row">Inherits Name Required:</th>
+            <td class="role-namerequired-inherited"> </td>
+          </tr>
+          <tr>
+            <th class="role-childpresentational-head" scope="row">Children Presentational:</th>
+            <td class="role-childpresentational"> </td>
+          </tr>
+          <tr>
+            <th class="role-presentational-inherited-head" scope="row">Inherits Presentational:</th>
+            <td class="role-presentational-inherited"> </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+    <div class="role" id="descriptionlist">
+      <rdef>descriptionlist</rdef>
+      <div class="role-description">
+        <p>A <rref>section</rref> containing <rref>descriptionitem</rref> and <rref>descriptionterm</rref> elements.</p>
+
+        <p>Description lists contain children whose <a>role</a> is <rref>descriptionitem</rref> and <rref>descriptionterm</rref> to represent a list of terms each having one or more descriptions.</p>
+
+        <p>Author requirements for <a>elements</a> whose <a>role</a> is <code>descriptionlist</code>:</p>
+        <ul>
+          <li>MUST only use an element with role <rref>descriptionterm</rref> as the first child in the description list.</li>
+          <li>Each element with role <rref>descriptionterm</rref> SHOULD have at least one following sibling element with role <rref>descriptionitem</rref>.</li>
```

Comment:
Since a description list must contain at least one `descriptionterm` and `descriptionitem`, and it must start with a `descriptionterm`, do we need this here?  

Or can it just be stated as part `descriptionterm`?  If we think this is necessary here, then should we mention that there MAY be multiple `descriptionitem` siblings to a single `descriptionterm` as well, or multiple `descriptionterm`s to a single or multiple `descriptionitem`s?

I think it'd be cleaner to just indicate those conditionals in their respective roles?